README.md

```diff
@@ -138,6 +139,12 @@ Type representing falsey values in TypeScript: `null | undefined | false | 0 | '
 
 [⇧ back to top](#table-of-contents)
 
+### mixed
+
+An arbitrary type that could be anything (same as `unknown`)
+
```

Comment:
On line 145, please add a link to the flow docs so it looks consistent with all the remaining flow types

Reply:
It still has wrong formatting, not showing below the description, please check how it's done in other types